app/studio/1.0.x/deploy-to-dev-portal.md

```diff
@@ -35,16 +34,16 @@ Kong Studio will prompt you for information to connect to your **Kong Admin API*
 * If your Admin API uses RBAC, enter the Token for to access the Workspace
 
 
-![Connect to Kong](https://doc-assets.konghq.com/studio/1.0/dev-portal/02-connect-to-kong.png)
+![Connect to Kong](https://s3.amazonaws.com/helpscout.net/docs/assets/59e383122c7d3a40f0ed78e2/images/5ea7fa1a2c7d3a7e9aebbe7d/file-aY1ixNgXnh.png)
 
-* Click **Connect to Kong** to validate Kong Studio’s connection. 
+* Click **Connect to Kong** to validate the connection. 
 
 
 ### Step 2. Deploy a Spec file
 
-If successful, Kong Studio will then prompt you to specify a file name for the spec file - this will be the name of the file in the Developer Portal.
+If successful, Insomnia Designer will then prompt you to specify a file name for the spec file - this will be the name of the file in the Developer Portal.
```

Comment:
```suggestion
If successful, Insomnia Designer will then prompt you to specify a name for the spec file &mdash; this will be the name that appears in the Developer Portal.
```